src/globals/scss/_helper-mixins.scss

```diff
@@ -36,10 +36,19 @@
   }
 }
 
-@mixin focus-outline {
-  outline: 1px dotted; // for non-webkit browsers
-  outline: auto 5px -webkit-focus-ring-color; // webkit default
-}
+@mixin focus-outline($type: 'border') {
+   @if ($type == 'border') {
+     outline: 1px solid $brand-01;
+   }
+
+   @if ($type == 'underline') {
```

Comment:
Added it because I thought it was needed for some components, but will make a new PR once all the other ones are merged in cleaning up the mixin (want to avoid merge conflicts).